packages/react-scripts/scripts/start.js

```diff
@@ -101,28 +101,19 @@ checkBrowsers(paths.appPath, isInteractive)
     const appName = require(paths.appPackageJson).name;
 
     const useTypeScript = fs.existsSync(paths.appTsConfig);
-    const tscCompileOnError = process.env.TSC_COMPILE_ON_ERROR === 'true';
```

Comment:
Is this no longer needed?

Reply:
not sure what the purpose of the flag is - eg. if it's to compile despite errors (not sure when that would make sense in a beginner project?)
The current behavior should be that the build breaks if there are type issues (if needed there are typescript specific comments to ignore ts in code)

Reply:
@iansu after reading the documentation for this flag my recommendation would be to remove it completely from v5 - as mentioned there are better alternatives (if you agree I'll remove the documentation for it too)

Reply:
@iansu I've removed the feature altogether and resolving the comment